nefertari/utils/data.py

```diff
@@ -94,3 +94,43 @@ def obj2dict(obj, classkey=None):
         return data
     else:
         return obj
+
+
+class FieldData(object):
```

Comment:
If fetching the field params from the event model is too much typing, I would add a convenience method on the event class.

But whatever, I don't care that much :-)